ServerCore/Program.cs

```diff
@@ -11,8 +14,23 @@ public static void Main(string[] args)
         }
 
         public static IWebHost BuildWebHost(string[] args) =>
-            WebHost.CreateDefaultBuilder(args)
-                .UseStartup<Startup>()
-                .Build();
+        WebHost.CreateDefaultBuilder(args)
+       //.ConfigureAppConfiguration((ctx, builder) =>
```

Comment:
Should this be cleaned up?

Reply:
I think it'll be recreated if I pick "add KeyVault" again in the future, so should be safe to delete.